crypto/ec/ecdh_ossl.c

```diff
@@ -66,6 +66,10 @@ int ecdh_simple_compute_key(unsigned char **pout, size_t *poutlen,
     BN_CTX_start(ctx);
     x = BN_CTX_get(ctx);
     y = BN_CTX_get(ctx);
+    if (y == NULL) {
```

Comment:
I was just giving an example.  People will be surprised, like you were, to not check both values.